apps/authentication/admin.py

```diff
@@ -0,0 +1,15 @@
+"""
+
+admin file
```

Comment:
Give an explanation on what is this file for. Please make sure you use correct grammatic. Check other files for guidance.